autoPyTorch/api/base_task.py

```diff
@@ -101,13 +105,13 @@ def send_warnings_to_log(
                     prediction,
                     np.sum(prediction, axis=1)
                 ))
-
-    if len(prediction.shape) < 1 or len(X_.shape) < 1 or \
-            X_.shape[0] < 1 or prediction.shape[0] != X_.shape[0]:
-        logger.warning(
-            "Prediction shape for model %s is %s while X_.shape is %s",
-            pipeline, str(prediction.shape), str(X_.shape)
-        )
+    if STRING_TO_TASK_TYPES.get(task_type, -1) != TIMESERIES_FORECASTING:
```

Comment:
why can't we use the same `if task in FORECASTING_TASKS`?

Reply:
And I don't want to make it inconsistent with the other tasks

Reply:
`task == TIMESERIES_FORECASTING` would have the same effect. I don't understand `We cannot get task before querying it`, could you explain?

Reply:
Okay, in the future we can also consider using `Enums`, but thanks for the explanation.